### PR TITLE
Research proposal prior to impelment Rust antelope.cdt

### PR DESCRIPTION
**Pull request by psy2848048**
### Project Abstract

Checking the most cost-effective plan among the cases we thought of is mandatory for building Rust Antelope CDT.

### For which grant [level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels) are you applying?

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 4 approvals

### Application Checklist

- [x] The [application template]([xxxx](https://github.com/eosnetworkfoundation/grant-framework/blob/main/applications/application-template.md)) has been copied, renamed (`my_project_name.md`) and updated.
- [x] The application file (`my_project_name.md`) has no spaces in its name, ends with the `.md` extension has been placed in the `applications` directory.
- [x] I have read and understood the:
  - [x] [FAQs](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/faq.md), 
  - [x] [application guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/grant_guidelines_per_category.md) and 
  - [x] [announcement guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/announcement-guidelines.md).
- [x] An EOS address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] This project has NOT been part of a token sale or NFT sale.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/eosnetworkfoundation/grant-milestones#process).

### How Did You Hear About our grants program?

- [ ] Social Media
- [ ] EOSNetwork.com
- [x] Personal Recommendation
- [ ] ENF Fireside Chat
- [ ] Investor/VC
- [ ] Online Search
- [ ] Hackathon
- [ ] Other: _______

### CODE REVIEW

**Review by DenisCarriere — APPROVED** [review]
Team is capable of delivering on grant scope of work.

Notes:
- Wasmer seems like a good option for VM integration
- To include in research as an active project, EOS Rust Contract Development Kit:
https://github.com/uuosio/rscdk

File: applications/cost-comparing-research-rust-cdt.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+### Milestone 1 — Research the difference between EOSVM and Wasmer
+
+- **Estimated duration:** 1 month
+- **FTE:**  2
+- **Costs:** 10,000 USD
+
+| ID  | Deliverable     | Specification                                                                                                            |
+|-----|-----------------|--------------------------------------------------------------------------------------------------------------------------|
+| 0a. | License         | MIT                                                                                                                      |
+| 1   | Research report | Comparison of language-LLVM compilation spec and WASM API spec of EOSVM and Wasmer<br/>A report and a follow-up proposal |
```
Comment (linh2931) [inline applications/cost-comparing-research-rust-cdt.md:165]: Can you add some details about what the report will contain? Like the structure, the approaches you take to compare, any code/experiment/sample contracts you run with Wasmer, how we can verify, and conclusions.

Comment (psy2848048) [reply applications/cost-comparing-research-rust-cdt.md]: Hi @linh2931 :)

The feedback is applied on the commit https://github.com/eosnetworkfoundation/grant-framework/pull/83/commits/ad50f04a16ec3160077af869e0780bbf26186eb3

**Review by linh2931 — APPROVED** [review]

**Comment by stephenpdeos** [comment]
All reviewers approved cc @tedcahalleos

**Pull request merged**